go.mod

```diff
@@ -5,11 +5,12 @@ go 1.14
 require (
 	github.com/RHsyseng/operator-utils v0.0.0-20200304191317-2425bf382482
 	github.com/coreos/prometheus-operator v0.40.0
+	github.com/cucumber/godog v0.11.0
 	github.com/go-openapi/spec v0.19.7
 	github.com/infinispan/infinispan-operator v0.0.0-20210106103300-03aa6d76d1b2
 	github.com/integr8ly/grafana-operator/v3 v3.4.0
 	github.com/keycloak/keycloak-operator v0.0.0-20200917060808-9858b19ca8bf
-	github.com/kiegroup/kogito-operator v0.12.1-0.20210319133405-929482d3d0aa
+	github.com/kiegroup/kogito-operator v0.12.1-0.20210323155633-ba32f4bb70a5
```

Comment:
Is this going to be updated with every product nightly? Or how often do we want to update his?

Reply:
It is not clear yet, maybe there will be a nightly job updating the version in case there is a new commit in community operator.
So far it is updated manually when needed.